examples/autocalibrate.py

```diff
@@ -0,0 +1,27 @@
+from octoliner import Octoliner
+
+
+# Create an object for working with 8-channel line sensor.
+octo = Octoliner()
+
+
+def main():
+    try:
+        while True:
+            print(
+                'Please enter the command "calc" to calculate the sensitivity',
+                'or Ctrl^C to exit.'
+            )
+            command = input()
+            if command == "calc":
+                # Optimal sensitivity calculation.
+                octo.optimize_sensitivity_on_black()
+                print("Optimal sensitivity: %.2f\n" % octo.get_sensitivity())
+            else:
+                print("Command is invalid")
```

Comment:
The “calc” command is somewhat complicated. Could we accept just Enter press and not consider the actual input at all?

Reply:
Done!